src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <p>Use cases:</p>
+        <ol>
+            <li>Send Packet Wrapper - Beginning of data packet (0).</li>
+            <li>Multiple RX Data (0x0C) messages until the entire bypass binary data is sent. Pay attention to the Acknowledge (0x14) message responded from GO if the GO Receive-Buffer is ready to accept the next RX Data (0x0C) message. The streaming flow control bit in the ACK is not relevant for this type of exchange and can be ignored.</li>
+            <li>Send Packet Wrapper - End of data packet (1).</li>
+            <li>At the end, GO sends confirmation with a Application Specific Data (0x1C) Type 0 (Modem transmission result) message to indicate if the packet has been accepted within 6 seconds.</li>
+        </ol>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0001</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Beginning of data packet
+                        <br></br>
+                        1 = End of data packet </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 2 - Request GO Device Data Message</h3>
+        <p>This message is used by an IOX which requires vehicle information from the GO device. The GO device responds with a GO Multi-Frame Data (0x27) - Type 2 message.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0002</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Message Version = 2</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 3 - Connect And Send Records</h3>
+        <p>This message requests the GO modem initiate a connection to the server.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0003</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Unused</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 4 - Request VIN Message</h3>
+        <p>An IOX uses this message to request the vehicle's VIN from the GO device. The GO device responds with a GO Multi-frame Data (0x27) - Type 3 message.</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on